ansible_galaxy/actions/install.py

```diff
@@ -79,8 +79,7 @@ def install_repositories_matching_repository_specs(galaxy_context,
     log.debug('already_installed_repository_spec_set: %s', already_installed_repository_spec_set)
 
     # This filters out already installed repositories unless --force. Aside from the warning, 'mazer install alikins.something_installed_already' is ok.
-    requirements_to_install = [y for y in requirements_list if y.requirement_spec not in already_installed_repository_spec_set and not force_overwrite]
-
+    requirements_to_install = [y for y in requirements_list if force_overwrite or y.requirement_spec not in already_installed_repository_spec_set]
```

Comment:
If you have a complex/multi-part conditional maybe a list comprehension isn't the most readable format to put that logic in, if we want to avoid future mishaps.

Reply:
Yeah, had the same thought. May just if/else on the 'force' before the comprehension.

That whole method needs revisit now that there is only one type of dep instead of 3. And now most of the objects involved are immutable/hashable so likely can collapse into some set ops.

Reply:
At some point, when the install bits is more 'transaction' oriented, it's likely the 'force' option will need to be set per collection (instead of overall), but that isn't the case now, so no reason not to mv the 'if force' outside.